doc/manual/configuration.rst

```diff
@@ -1195,3 +1200,11 @@ Specifies alias names for packages::
 
    alias:
       myApp: "host/files/group/app42"
+
+build-args
```

Comment:
I'd rather find dedicated configuration options more elegant. Something like
```yaml
config:
  dev:
    mode: "build-only"
    logfiles: False
```

They would be taken as defaults that can be overridden by the various command line options.